apps/web/src/components/AccessRisk/index.tsx

```diff
@@ -11,17 +10,18 @@ import {
   RefreshIcon,
   Tag,
   Text,
-  useTooltip,
   promotedGradient,
-  AutoRenewIcon,
+  useTooltip,
 } from '@pancakeswap/uikit'
+import isUndefinedOrNull from '@pancakeswap/utils/isUndefinedOrNull'
+import AccessRiskTooltips from 'components/AccessRisk/AccessRiskTooltips'
+import { fetchRiskToken } from 'components/AccessRisk/utils/fetchTokenRisk'
 import { useEffect, useMemo, useState } from 'react'
-import { useUserTokenRisk } from 'state/user/hooks/useUserTokenRisk'
 import { useAllLists } from 'state/lists/hooks'
+import { useUserTokenRisk } from 'state/user/hooks/useUserTokenRisk'
 import { styled } from 'styled-components'
 import useSWRImmutable from 'swr/immutable'
-import { fetchRiskToken } from 'components/AccessRisk/utils/fetchTokenRisk'
-import AccessRiskTooltips from 'components/AccessRisk/AccessRiskTooltips'
+import { ACCESS_TOKEN_SUPPORT_CHAIN_IDS } from 'views/Swap/SwapFeaturesContext'
```

Comment:
Let's move this constant out from Swap views if it's used somewhere else. Let's put it under `src/constants`

Reply:
maybe add to `src/config/constants/supportedChains`

Reply:
Done. Move under `AccessRisk`